yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts

```diff
@@ -17,8 +16,9 @@ import {
   TokenPortalBytecode,
 } from '@aztec/l1-artifacts';
 import { Chain, HttpTransport, PublicClient, getContract } from 'viem';
-import { setup } from './setup.js';
+import { delay, deployNonNativeL2TokenContract, setup } from './utils.js';
```

Comment:
How come the non-native is inside the utils? Don't seem particular generic, is it because it is reused in here and in the new integration?